FRW/UI/Component/HalfSheetModal.swift

```diff
@@ -60,6 +60,25 @@ struct SheetHeaderView: View {
     }
 }
 
+@available(iOS 16, *)
+struct HalfSheetModifier: ViewModifier {
+    @State var height: CGFloat = 0
+    
+    func body(content: Content) -> some View {
+        content
+            .fixedSize(horizontal: false, vertical: true)
+            .getHeight($height)
+            .presentationDetents([.height(height), .large])
```

Comment:
`height()` is available on iOS 16 only, that's the reason why this fix doesn't work on older versions